beacon_node/http_api/src/ui.rs

```diff
@@ -71,6 +71,79 @@ pub fn get_validator_count<T: BeaconChainTypes>(
     })
 }
 
+#[derive(PartialEq, Serialize, Deserialize)]
+pub struct ValidatorInfoRequestData {
+    indices: Vec<u64>,
+}
+
+#[derive(PartialEq, Serialize, Deserialize)]
+pub struct ValidatorInfoValues {
+    epoch: u64,
+    total_balance: u64,
+}
```

Comment:
We should probably quote these `u64`s for JS-safety. Seeing as the current UI works with them unquoted I think we can come back for this.